src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java

```diff
@@ -171,7 +172,7 @@ public String getEntityTypeAsString() {
 
     @Override
     public String getBackupIdentifier() {
-        return "Account";
+        return Const.SystemParams.ACCOUNT_ATTRIBUTES_BACKUP_LOG_MSG + this.googleId;
```

Comment:
Beside `AccountAttributes.java`, there are more `*Atrributes.java` need to be changed.